admin/Public/A2B_trunk_report.php

```diff
@@ -298,6 +298,7 @@
                 if ($i % 2 == 0) {
                     $bgcolor = "bgcolor='#F2F2EE'";$mouseout = "bgColor='#F2F2EE'";} else {$bgcolor = "bgcolor='#FCFBFB'";$mouseout = "bgColor='#FCFBFB'";
                 }
+                $ALOC = $ALOC ?? 0;
```

Comment:
This is already initialized to 0 at beginning of script. If it's null down here, it must be because of assignment at line 113. Better would be to assign 0 in case of null there, i.e. `$ALOC = $val[0] ?? 0;`